RHIDP-979: Adding YAMl config details to the Helm upgrade procedure (#139)
redhat-developer/red-hat-developers-documentation-rhdh: getting-started/proc-add-custom-app-file-openshift.adoc

. Go to the *Helm* tab.
+
The list of Helm Releases appears on the page.
. Click the three dots corresponding to a Helm release and select *Upgrade*.
. Use one of the following views to edit the Helm configuration:

** Using *Form view* 
+
.. Expand *Root Schema → Backstage chart schema → Backstage parameters → Extra app configuration files to inline into command arguments*. 
.. Click the *Add Extra app configuration files to inline into command arguments* link.
.. Enter the value in the following fields:
+
--
* *configMapRef*: `app-config-rhdh`
* *filename*: `app-config-rhdh.yaml`
--
.. Click *Upgrade*.

** Using *YAML view* 

> .. Set the value of the following parameters:

Reviewer: This visualization is not easy to understand as the YAML view shows the full manifest of the Helm Chart, so it should be better showing the yaml structure of this change. For example:

```yaml
# ... other Developer Hub Helm Chart configuration
upstream:
  backstage:
    extraAppConfig:
      - configMapRef: app-config-rhdh
        filename: app-config-rhdh.yaml
# ... other Developer Hub Helm Chart configuration
```

. Go to the *Helm* tab.
+
The list of Helm Releases appears on the page.

> . Click the three dots corresponding to a Helm release and select *Upgrade*.

Reviewer 2: IBM style guide says to refer to this icon by its tooltip or as an _overflow menu_.

> . Click the three dots corresponding to a Helm release and select *Upgrade*.

Reviewer 2: ```suggestion
. Click the overflow menu on a Helm release and select *Upgrade*.
```

. Use one of the following views to edit the Helm configuration:

** Using *Form view* 
+
.. Expand *Root Schema → Backstage chart schema → Backstage parameters → Extra app configuration files to inline into command arguments*. 
.. Click the *Add Extra app configuration files to inline into command arguments* link.
.. Enter the value in the following fields:
+
--
* *configMapRef*: `app-config-rhdh`
* *filename*: `app-config-rhdh.yaml`
--
.. Click *Upgrade*.

> ** Using *YAML view* 

Reviewer 2: [This section](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/135/files#diff-2e8740a4b09b45b2f17d1753bd5040018a1b32d267e415ff37f7da0997ff3f3fR29) of PR #135 writes about the Form and YAML views a little differently. At some point, we should ensure we are presenting the Form and YAML configuration instructions consistently across the entire RHDH doc set.

Author: Yes, @linfraze. 
What i noticed in the PR#135: if you read line no 29-31, you find lesser information and more of text and redundant phrases. for example, `to configure the {product-short} instance`, same phrase we have repeated in the sub-bullet points. IMO, the way we have introduced form view and YAML view config in this PR is to the point and more catchy and visible.

Author: In addition to this, using the YAML code snippet provides better clarity and helps in understanding the correct placement of parameters and values within a long YAML code.
